### ISSUE TITLE
add prepDE.py

### ISSUE THREAD

**Issue by zlskidmore** [genome/docker-rnaseq#8]
Adding note to myself or whoever get's to it first, assuming we want this of course :). The genome/analysis workflows currently don't output raw expression counts necessary for DE analysis with bioc packages edgeR and deseq2. We should add the stringtie python [script](http://ccb.jhu.edu/software/stringtie/dl/prepDE.py) in this VM so users can do this conversion.

Possibly we should modify the RNA workflow to just output those raw results as well.

**Comment by chrisamiller** [comment]
Yeah, those outputs are present in the kallisto results, do not exist for stringtie.  We would welcome a PR that adds that step!
